r124/u_inref/ydb312_gtm8182g.csh

```diff
@@ -0,0 +1,95 @@
+#!/usr/local/bin/tcsh -f
+#################################################################
+#								#
+# Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	#
+# All rights reserved.						#
+#								#
+#	This source code contains the intellectual property	#
+#	of its copyright holder(s), and is made available	#
+#	under a license.  If you do not know the terms of	#
+#	the license, please stop and do not read further.	#
+#								#
+#################################################################
+#
+setenv gtm_repl_instance "mumps.repl"
+unsetenv gtm_db_counter_sem_incr # avoid semaphore counter overflow
+
+#setenv ydb_gbldir "GBL3"
+#setenv gtmgbldir $ydb_gbldir
+#DB3 will be the main DB with its global dir containing DB1 and DB2
+
+$MULTISITE_REPLIC_PREPARE 2
+
+echo "Create the DB"
+# create a 2 region DB with mumps.dat tied to DEFAULT and a.dat tied to AREG
+$gtm_tst/com/dbcreate.csh mumps 2 -gld_has_db_fullpath >>& dbcreate.out
+if ($status) then
+	echo "DB Create Failed, Output Below"
+	cat dbcreate.out
+	exit -1
+endif
+echo ""
+echo ""
+
+#C[HANGE] -R[EGION] region-name [-REGION-qualifier...]
+
+#Create the global directory for DB1 and point its GLD towards DB3's mumps.dat DB file
+#  add -segment cusseg -file=cus.dat
+#  C[HANGE] -S[EGMENT] segment-name [-SEGMENT-qualifier...]
+
+setenv ydb_gbldir "GBL1"
```

Comment:
Remove any ydb_gbldir usages. Just gtmgbldir usages are fine since that is what the test framework works with.